app/src/main/java/com/example/bottomnav/Parse.java

```diff
@@ -71,6 +100,17 @@ private void storeStuct(IASTDeclaration[] declarations, String name) throws Exce
         structRepo.put(name, struct);
     }
 
+    public ArrayList getAssociatedStruct(String IDName) {
```

Comment:
Style: convention for variable names tend to be camelCase, starting with lower case - so use `idName` here - and similarly `idStruct` above